app/models/concerns/patient_reportable.rb

```diff
@@ -25,18 +25,14 @@ module PatientReportable
         .distinct(:patient_id)
     end
 
-    scope :for_reports, ->(with_exclusions: false, exclude_ltfu_as_of: nil) do
-      if with_exclusions
-        if exclude_ltfu_as_of
-          with_hypertension
-            .excluding_dead
-            .not_ltfu_as_of(exclude_ltfu_as_of)
-        else
-          with_hypertension
-            .excluding_dead
-        end
+    scope :for_reports, ->(exclude_ltfu_as_of: nil) do
+      if exclude_ltfu_as_of
+        with_hypertension
+          .excluding_dead
+          .not_ltfu_as_of(exclude_ltfu_as_of)
       else
         with_hypertension
+          .excluding_dead
```

Comment:
This would be more clear as something like this:

```ruby
 scope :for_reports, ->(exclude_ltfu_as_of: nil) do
    query = with_hypertension.excluding_dead
    query.not_ltfu_as_of(exclude_ltfu_as_of) if exclude_ltfu_as_of
    query
end
```

Reply:
Maybe we do `exclude_ltfu_as_of.end_time` if all callers are consistently passing in a Period?  Or we could gate that on a `respond_to?` to be safe.

Reply:
The `end_time` is a query-specific clause in the two places `exclude_ltfu_as_of` used currently. "Which LTFU patients does the query want to remove?". I think we should let the query make that decision, keeps the scope decoupled.